### PR TITLE
{tools}[GCCcore/10.2.0,GCCcore/10.3.0,GCCcore/11.2.0] xxd v8.2.4220

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by sassy-crick** [comment]
This is related to issue [11209](https://github.com/easybuilders/easybuild-easyconfigs/issues/11209) and needed for `Plumed` installations. This does not install `vim` but only the `xxd` part of `vim`.

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
eb-u2004 - Linux UNKNOWN UNKNOWN, x86_64, AMD Ryzen 9 3900X 12-Core Processor (zen2), Python 3.8.10
See https://gist.github.com/73411af9cb9c47db3cef4af4f78dc355 for a full test report.

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
eb-u2004 - Linux Ubuntu 20.04, x86_64, AMD Ryzen 9 3900X 12-Core Processor (zen2), Python 3.8.10
See https://gist.github.com/2b7ffbbe38bfcfd47e11ea525ff5fa80 for a full test report.

**Comment by zao** [comment]
You seem to be missing the `binutils` build dependency customary when using GCCcore.

**Comment by sassy-crick** [comment]
Apologies, that slipped my attention. I have added that now. Thanks for pointing it out!

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

File: easybuild/easyconfigs/x/xxd/xxd-8.2.4220-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# uploaded by J. Sassmannshausen
+
+easyblock = 'MakeCp'
+
+name = 'xxd'
+version = '8.2.4220'
+
+homepage = 'https://www.vim.org'
+description = """xxd is part of the VIM package and this will only install xxd, not vim!
+xxd converts to/from hexdumps of binary files."""
```
Comment (lexming) [inline easybuild/easyconfigs/x/xxd/xxd-8.2.4220-GCCcore-10.2.0.eb:12]: The description is used in the module file, so it should be relevant to the final user. I would remove any mentions to what will be installed
```suggestion
description = "xxd is part of the VIM package and serves to convert to/from hexdumps of binary files."
```

**Review by lexming — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3112.skitty.os - Linux UNKNOWN UNKNOWN, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/927a5e115ea197eeab1cb05b1088dc6c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3526.doduo.os - Linux UNKNOWN UNKNOWN, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/03dd8b8829c7067f342501375f22fddb for a full test report.

**Comment by boegel** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**